styles.css

```diff
@@ -0,0 +1,170 @@
+body {
+    margin: 15;
+    font-family: Arial, Holvetica ;
+    background-image: linear-gradient(to bottom right,#D0F1F5, #D0F1F5,#A3DCE2, #1CD0E4,#52C1CD)
+  }
+  .topnav {
+    overflow: hidden;
+    background-color: #333;
+  }
+  .topnav a {
+    float: left;
+    display: block;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+  }
+  .topnav a:hover {
+    background-color: #ddd;
+    color: black;
+  }
+  .active {
+    background-color: #4CAF50;
+    color: white;
+  }
+  .topnav .icon {
+    display: none;
+  }
+  .topnav.responsive a {
+    float: none;
+    display: block;
+    right: 0;
+    text-align: left;
+  }
+  
+  @media screen and (max-width: 600px) {
+    .topnav.responsive {position: relative;}
+    .topnav.responsive .icon {
+      position: absolute;
+      right: 0;
+      top: 0;
+    }
+  }
+  
+  @media screen and (max-width: 600px) {
+    .topnav a:not(:first-child) { display: none;}
+    .topnav a.icon {
+      float: right;
```

Comment:
`Float` is not advisable to use anymore 😢 
Try to manipulate all layout with `Flexbox` or `Grid` 🙏🏽